components/SearchBar/styles.js

```diff
@@ -0,0 +1,33 @@
+import styled from "styled-components";
+
+export const SearchBarContainer = styled.form`
+  display: flex;
+  width: 296px;
+  height: 17px;
```

Comment:
I would be very careful about setting manual heights to elements. It is generally (not always) something we want to avoid. I think if you were to remove this we would not see any difference.